components/brave_today/browser/brave_news_p3a.cc

```diff
@@ -143,8 +219,13 @@ void RecordTotalCardViews(PrefService* prefs,
   RecordToHistogramBucket(kTotalCardViewsHistogramName, buckets, total);
 }
 
-void RecordAtStart(PrefService* prefs) {
+void RecordAtInit(PrefService* prefs) {
   ResetCurrSessionTotalViewsCount(prefs);
+
+  RecordLastUsageTime(prefs);
+  RecordNewUserReturning(prefs);
```

Comment:
Could you add a comment to document how "new user" status is tracked. Either in `RegisterPrefs` or in one of the functions doing the recording. I'm unclear what the definition is, and how the "I'm a first-time Brave News user this week" part is implemented.

Is the idea to always return `0` after the first week of session usage?

Reply:
I added some more comments in the `RecordNewUserReturning` method. Hopefully it's more clear.

After the first week of usage, the metric value would be `1`.

Reply:
Ok, Thanks.